Dockerfile

```diff
@@ -1,21 +1,12 @@
-FROM ocrd/core-cuda
+ARG DOCKER_BASE_IMAGE
+FROM $DOCKER_BASE_IMAGE
 ARG VCS_REF
 ARG BUILD_DATE
-MAINTAINER unixprog@gmail.com
-LABEL maintainer="https://ocr-d.de"
-LABEL org.label-schema.vendor="DFG-Funded Initiative for Optical Character Recognition Development"
-LABEL org.label-schema.name="ocrd_kraken"
-LABEL org.label-schema.vcs-ref=$VCS_REF
-LABEL org.label-schema.vcs-url="https://github.com/OCR-D/ocrd_kraken"
-LABEL org.label-schema.build-date=$BUILD_DATE
-LABEL org.opencontainers.image.vendor="DFG-Funded Initiative for Optical Character Recognition Development"
-LABEL org.opencontainers.image.title="ocrd_kraken"
-LABEL org.opencontainers.image.description="Kraken bindings"
-LABEL org.opencontainers.image.source="https://github.com/OCR-D/ocrd_kraken"
-LABEL org.opencontainers.image.documentation="https://github.com/OCR-D/ocrd_kraken/blob/${VCS_REF}/README.md"
-LABEL org.opencontainers.image.revision=$VCS_REF
-LABEL org.opencontainers.image.created=$BUILD_DATE
-LABEL org.opencontainers.image.base.name=ocrd/core-cuda
```

Comment:
Why remove all those?

These are conforming to the newer labelling standard, so all of our images should adopt them in the future. (I completely forgot about this.)

Reply:
Thanks, I have put them back in